Eatery/Controllers/Eateries/ImageParallaxScrollViewController.swift

```diff
@@ -104,6 +106,15 @@ class ImageParallaxScrollViewController: UIViewController {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
 
+        // These variables are placed here rather than in CampusMenuVC because
+        // when testing, navBarHeight gave 148 sometimes for some reason...
+        // I'm not sure why this happened but it was 100% correct when put in here
```

Comment:
could be that it was getting the navBarHeight of the previous screen since here you're guaranteed the views appeared before running the code - maybe find out what code changes the navigationBar's height and put that in viewWillAppear in this file? would be ideal to move these variables to CampusMenuVC if they're unused in this file to have everything in the appropriate place